circleci: failed tests (13427): TestSSLEnforcement

The following test appears to have failed:

[#13427](https://circleci.com/gh/cockroachdb/cockroach/13427):

```
E160225 20:40:22.380514 server.go:2159  http: TLS handshake error from 127.0.0.1:50645: tls: first record does not look like a TLS handshake
E160225 20:40:22.386400 server.go:2159  http: TLS handshake error from 127.0.0.1:50646: tls: first record does not look like a TLS handshake
E160225 20:40:22.390833 server.go:2159  http: TLS handshake error from 127.0.0.1:50647: tls: first record does not look like a TLS handshake
E160225 20:40:22.393719 server.go:2159  http: TLS handshake error from 127.0.0.1:50648: tls: first record does not look like a TLS handshake
I160225 20:40:22.407096 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- FAIL: TestSSLEnforcement (0.28s)
	authentication_test.go:148: [28]: expected success=true, got err=Post https://127.0.0.1:54368/sql/Execute: stream error: stream ID 13; STREAM_CLOSED
=== RUN   TestParseStores
I160225 20:40:22.408981 server/context.go:165  can't read available memory from cgroups (open /sys/fs/cgroup/memory/memory.limit_in_bytes: no such file or directory)
	setting default rocksdb cache size to 30 GiB (half of system memory)
I160225 20:40:22.409142 server/context.go:165  can't read available memory from cgroups (open /sys/fs/cgroup/memory/memory.limit_in_bytes: no such file or directory)
	setting default rocksdb cache size to 30 GiB (half of system memory)
I160225 20:40:22.409247 server/context.go:225  1 storage engine(s) specified
I160225 20:40:22.409412 server/context.go:165  can't read available memory from cgroups (open /sys/fs/cgroup/memory/memory.limit_in_bytes: no such file or directory)
	setting default rocksdb cache size to 30 GiB (half of system memory)
I160225 20:40:22.409457 server/context.go:225  2 storage engine(s) specified
--
W160225 20:40:30.740807 kv/dist_sender.go:609  failed to invoke Merge [/System/"tsd!cr.node.pgwire.conns\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.pgwire.bytesin\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.pgwire.bytesout\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1m-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1m-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1m-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1m-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1m-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1m-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1m-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1m-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-10m-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-10m-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-10m-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-10m-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-10m-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-10m-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-10m-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-10m-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.txn.commit.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.select.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.update.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.insert.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.delete.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.ddl.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.misc.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1h-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1h-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1h-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1h-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1h-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1h-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1h-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.latency-1h-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.txn.begin.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.txn.abort.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.sql.txn.rollback.count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.commits-10m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.commits-count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1m-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1m-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1m-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1m-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1m-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1m-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1m-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1m-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.aborts-1m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.aborts-count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.abandons-10m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.restarts-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.restarts-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.restarts-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.restarts-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.restarts-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.restarts-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.restarts-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.restarts-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.aborts-10m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.aborts-1h\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.commits-1m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.abandons-1m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-10m-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-10m-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-10m-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-10m-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-10m-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-10m-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-10m-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-10m-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.commits-1h\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.abandons-1h\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.abandons-count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1h-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1h-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1h-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1h-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1h-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1h-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1h-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.txn.durations-1h-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.error-10m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.error-count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1h-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1h-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1h-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1h-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1h-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1h-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1h-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1h-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.success-1m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.success-1h\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.success-count\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.error-1m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1m-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1m-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1m-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1m-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1m-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1m-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1m-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-1m-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-10m-max\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-10m-p99.999\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-10m-p99.99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-10m-p99.9\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-10m-p99\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-10m-p90\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-10m-p75\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.latency-10m-p50\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.success-10m\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.node.exec.error-1h\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.lastupdatenanos\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.intentbytes\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.livecount\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.intentcount\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.ranges.available\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.intentage\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.keybytes\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.valcount\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.gcbytesage\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.ranges\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.ranges.leader\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.ranges.replicated\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.capacity\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.capacity.available\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.livebytes\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.valbytes\x00\x01\x89\xf8\x06,T1",/Min), Merge [/System/"tsd!cr.store.keycount\x00\x01\x89\xf8\x06,T1",/Min): failed to send RPC: too many errors encountered (1 of 1 total): server/node.go:594: node 1 stopped
I160225 20:40:30.740856 client/db.go:488  failed batch: failed to send RPC: too many errors encountered (1 of 1 total): server/node.go:594: node 1 stopped
W160225 20:40:30.740893 ts/db.go:96  error writing time series data: failed to send RPC: too many errors encountered (1 of 1 total): server/node.go:594: node 1 stopped
I160225 20:40:30.741153 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestMetricsRecording (0.17s)
FAIL
FAIL	github.com/cockroachdb/cockroach/server	10.087s
=== RUN   TestMetricsRecorder
--- PASS: TestMetricsRecorder (0.01s)
=== RUN   TestRuntimeStatRecorder
I160225 20:40:24.698910 server/status/runtime.go:137  runtime stats: 4 goroutines, 7.01MiB active, 50000000.00cgo/sec, 370530.00/185260.00 %(u/s)time, 10307.88 %gc (4x)
--- PASS: TestRuntimeStatRecorder (0.00s)
PASS
ok  	github.com/cockroachdb/cockroach/server/status	0.050s
=== RUN   TestSplitOrExpr
--- PASS: TestSplitOrExpr (0.01s)
=== RUN   TestSplitAndExpr

```
Please assign, take a look and update the issue accordingly.

--- a/server/authentication_test.go
+++ b/server/authentication_test.go
@@ -63,7 +63,6 @@ func sqlForUser(context *base.Context) proto.Message {
 // Verify client certificate enforcement and user whitelisting.
 func TestSSLEnforcement(t *testing.T) {
 	defer leaktest.AfterTest(t)
-	t.Skip("#4663")
 	s := StartTestServer(t)
 	defer s.Stop()
 

